orchagent/fgnhgorch.cpp

```diff
@@ -968,20 +1085,22 @@ bool FgNhgOrch::computeAndSetHashBucketChanges(FGNextHopGroupEntry *syncd_fg_rou
                 (bank_member_changes[bank_idx].nhs_to_add.size() != 0 &&
                  bank_member_changes[bank_idx].nhs_to_del.size() != 0))
         {
+            SWSS_LOG_INFO("active nhs in bank %u", bank_idx);
             /* Active bank is is determined by there being active nhs on the bank OR
              * an edge case where all active_nhs went down(nhs_to_del > 0) BUT
-             * simultaneously, nhs were added(nhs_to_add > 0). 
+             * simultaneously, nhs were added(nhs_to_add > 0).
              * Route this to fn which deals with active banks
              */
-            if (!setActiveBankHashBucketChanges(syncd_fg_route_entry, fgNhgEntry, 
-                        bank_idx, bank_idx, bank_member_changes, nhopgroup_members_set, ipPrefix))
+            if (!setActiveBankHashBucketChanges(syncd_fg_route_entry, fgNhgEntry,
+                        bank_idx, bank_member_changes[bank_idx], nhopgroup_members_set, ipPrefix))
             {
                 return false;
             }
         }
         else
         {
-            if (!setInactiveBankHashBucketChanges(syncd_fg_route_entry, fgNhgEntry, 
+            SWSS_LOG_INFO("No active nhs in bank %u", bank_idx);
```

Comment:
Log exists in setInactiveBankHashBucketChanges, can we remove this?

Reply:
Removed.

Reply:
Sorry, removed it now.